lib/stripe/util.rb

```diff
@@ -208,6 +209,14 @@ def self.normalize_opts(opts)
       end
     end
 
+    # Will either retrieve the contents of a URL or return the file_param
+    # if the argument does not satisfy the regex
+    def self.normalize_file_params(file_param)
+      return file_param unless file_param.match?(/^http.*/)
+
+      open(file_param)
+    end
```

Comment:
Opted for a _very_ basic regex here. The URI library for parsing and using the normal regex is a bit in flux between Ruby 2.3 and current versions.